extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/CalciteSelectQueryMSQTest.java

```diff
@@ -174,4 +174,25 @@ public void testArrayAggQueryOnComplexDatatypes()
       );
     }
   }
+
+  @Ignore
+  @Override
+  public void testEquiJoin2()
+  {
+
+  }
+
+  @Ignore
+  @Override
+  public void testEquiJoin3()
+  {
+
+  }
+
+  @Ignore
+  @Override
+  public void testEquiJoin4()
+  {
+
+  }
```

Comment:
1. Can you add a comment on why these joins are not working with MSQ? Per our knowledge, MSQ supports joins and hence the inability to work with one is concerning and should be documented.
2. There's a `msqIncompatible` flag that you can use instead, which is for this purpose itself unless this error happens in the planning phase.